monai/data/wsi_reader.py

```diff
@@ -270,11 +271,13 @@ class WSIReader(BaseWSIReader):
     def __init__(self, backend="cucim", level: int = 0, channel_dim: int = 0, **kwargs):
         super().__init__(level, channel_dim, **kwargs)
         self.backend = backend.lower()
-        self.reader: Union[CuCIMWSIReader, OpenSlideWSIReader]
+        self.reader: Union[CuCIMWSIReader, OpenSlideWSIReader, TiffFileWSIReader]
         if self.backend == "cucim":
             self.reader = CuCIMWSIReader(level=level, channel_dim=channel_dim, **kwargs)
         elif self.backend == "openslide":
             self.reader = OpenSlideWSIReader(level=level, channel_dim=channel_dim, **kwargs)
+        elif self.backend == "tifffile":
+            self.reader = TiffFileWSIReader(level=level, channel_dim=channel_dim, **kwargs)
         else:
             raise ValueError(f"The supported backends are cucim and openslide, '{self.backend}' was given.")
```

Comment:
raise ValueError(f"The supported backends are cucim, openslide and tifffile ....